tests/codegen_testcases/solidity/global_strings.sol

```diff
@@ -0,0 +1,153 @@
+// RUN: --target solana --emit llvm-ir
+// READ: ServiceRegistrySolana.ll
+
+
+// BEGIN-CHECK: "math overflow,\0A"
+// NOT-CHECK: "math overflow,\0A"
+
+import "solana";
+contract ServiceRegistrySolana {
+    struct Service {
+        address serviceOwner;
+        uint64 securityDeposit;
+        address multisig;
+        bytes32 configHash;
+        uint32 threshold;
+        uint32 maxNumAgentInstances;
+        uint32 numAgentInstances;
+        uint8 state;
+        uint32[] agentIds;
+        uint32[] slots;
+        uint64[] bonds;
+        address[] operators;
+        address[] agentInstances;
+        uint32[] agentIdForAgentInstances;
+    }
+
+    address public owner;
+    address public escrow;
+    string public baseURI;
+    uint32 public totalSupply;
+    string public constant CID_PREFIX = "f01701220";
+    uint64 public slashedFunds;
+    address public drainer;
+    string public constant VERSION = "1.0.0";
+    mapping(bytes32 => uint64) public mapOperatorAndServiceIdOperatorBalances;
+    mapping (address => address) public mapAgentInstanceOperators;
+    mapping (address => bool) public mapMultisigs;
+    Service public services;
+
+
+
+    constructor(address a, string memory _baseURI)
+    {
```

Comment:
Does the contract need all those empty functions for this case?

Reply:
I can shorten the contract, but I left it as it is because it used to generate 1008 definitions of `math overflow`.
What is your opinion in shortening it?

Reply:
Well, it's not the end of the world. I'll leave that up to you. I just thought that if in the end only 1 single `math overflow` string instead of 1008 should be generated, the test code could be more to the point.